package/kernel/linux/modules/video.mk

```diff
@@ -387,6 +387,65 @@ endef
 
 $(eval $(call KernelPackage,drm-radeon))
 
+define KernelPackage/gpu-lima
+  SUBMENU:=$(VIDEO_MENU)
+  TITLE:=Mali-4xx GPU support
+  DEPENDS:=+kmod-drm
+  KCONFIG:= \
+	CONFIG_DRM_VGEM \
+	CONFIG_DRM_GEM_CMA_HELPER=y \
+	CONFIG_DRM_LIMA
+  FILES:= \
+	$(LINUX_DIR)/drivers/gpu/drm/vgem/vgem.ko \
+	$(LINUX_DIR)/drivers/gpu/drm/scheduler/gpu-sched.ko \
+	$(LINUX_DIR)/drivers/gpu/drm/lima/lima.ko
+  AUTOLOAD:=$(call AutoProbe,lima vgem)
+endef
```

Comment:
`DEPENDS:=@TARGET_rockchip` would be a good idea, as other platforms most likely won't benefit from this (at least at the moment)